### PR TITLE
Colony is now working.

### PR DESCRIPTION
**Pull request by hdavidzhu**
Writing up a README in the wiki right now.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -89,7 +89,13 @@ gulp.task('nodemon', function () {
   })
 });
 
-gulp.task('watch', function () {
+// Visualize the folder structures.
```
Comment (sihrc) [inline gulpfile.js:92]: I don't think we should mix non-production code with our production code.

Comment (hdavidzhu) [inline gulpfile.js:92]: Where should I place this thing? I can't find a good home for it elsewhere.

Comment (sihrc) [inline gulpfile.js:92]: Make it a shell script and put it in scripts?

Comment (sihrc) [inline gulpfile.js:92]: We can rename codeship -> scripts and do more compartmentalization there.

**Comment by hdavidzhu** [comment]
noooooo I didn't rebase yet - was going to do that after the thumbs up

**Pull request merged**